src/cogent3/app/sample.py

```diff
@@ -832,6 +832,44 @@ def __init__(self, gc=1):
         -------
         A new sequence collection, or False if not all the named sequences are
         in the collection.
+
+        Examples
+        --------
+        Removes trailing stop codons from sequences in a sequence collection or alignment
+
+        >>> from cogent3 import make_unaligned_seqs, make_aligned_seqs, get_app
+
+        Create a sample unaligned sequence data and an app to remove stop codons
```

Comment:
Replace `Create a sample unaligned sequence data` with `Create a sample of unaligned sequences`